### ISSUE TITLE
Create new export process page

### ISSUE THREAD

**Issue by naushinthomson** [elifesciences/schematron-wiki#222]
- [ ] Create page covering process of exporting articles from eJP to Exeter/Kriya
- [ ] Add description of what is being covered
- [ ] Add description of checks required on this content, compiled from current protocol documents
- [ ] Add examples covering correct and incorrect scenarios, as required to support checks
- [ ] Populate with schematron messages
- [ ] Provide explanations for how to fix schematron errors and warnings
- [ ] Review house style document to ensure all requirements are covered by GitBook and schematron
- [ ] Review schematron messages related to this topic to ensure they are user friendly
- [x] Fred approved page
- [x] James approved page
- [x] Naushin approved page
- [x] Rebecca approved page
- [ ] Share page with Exeter and invite feedback

### Definition of done
- [ ] Page revised after team comments
- [ ] Team reviewed and approved page?
- [ ] Assign Fred to update the Schematron

**Comment by fred-atherden** [comment]
@naushinthomson, @rkcook312, @JGilbert-eLife, the first draft of this page is up: https://elifeproduction.slab.com/posts/vor-export-workflow-yqbyiovo. 

Please take a look when you have a chance. 

I wasn't sure if it was worth covering some of the eJP processes more in this page (or if that's best captured elsewhere). If so, @JGilbert-eLife it would be good to have your assistance with populating that section.

**Comment by fred-atherden** [comment]
@gnott, we document all our processes in Production in a tool called Slab. This ticket represents adding a page to our protocols which details the accepted submission workflow. The draft of that page is up here: https://elifeproduction.slab.com/posts/vor-export-workflow-yqbyiovo.

If you have time, it'd be great to get your eyes on it, as there will no doubt be things I've missed and/or processes I haven't documented quite rightly.

I'm not sure if you'll have access to the link above. If not, [here's a PDF copy](https://github.com/user-attachments/files/16801519/VOR.export.workflow.-.Slab.pdf). If you have a chance, feel free to feedback in whatever way is easiest (comments here, in slack, a marked up PDF).

**Comment by rkcook312** [comment]
Hi @fred-atherden, had a chance to review, looks good to me, are there instances which have repeated before causing the export process to fail / delay, could these be captured here and guidance how to solve?

**Comment by fred-atherden** [comment]
Good point - thanks @rkcook312. I've added a new section - 'Failures'. Please take a look and LMK if you have any feedback

**Comment by rkcook312** [comment]
Thanks @fred-atherden, re-read and all good to me, no further comments at the moment

**Comment by naushinthomson** [comment]
@fred-atherden Looks good to me - I've just left one comment!

**Comment by JGilbert-eLife** [comment]
Looks great, @fred-atherden !

**Comment by fred-atherden** [comment]
@naushinthomson, anything else needed here? I don't think this needs sharing w/ Exeter. Shall I close?

**Comment by fred-atherden** [comment]
Thanks @naushinthomson, can I eave you to publish the page in Slab?
